### PR TITLE
Fixed references in README to Sender documentation

### PR DESCRIPTION
**Pull request by jhmcstanton**
Removed a link that does not seem to have a matching section in the new sender documentation.

Corrected a link to the sender documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,9 +167,9 @@ See [Instantiating ApplicationTags](https://github.com/wavefrontHQ/wavefront-sdk
 
 A `WavefrontSender` object implements the low-level interface for sending data to Wavefront. You can choose to send data using the [Wavefront proxy](https://docs.wavefront.com/proxies.html) or [direct ingestion](https://docs.wavefront.com/direct_ingestion.html).
 
-* If you have already set up a `WavefrontSender` for another SDK that runs in the same process, use that one. (For details about sharing a Wavefront sender, see [Share a Wavefront Sender](https://github.com/wavefrontHQ/wavefront-sdk-python/blob/master/docs/sender.md#share-a-wavefront-sender).
+* If you have already set up a `WavefrontSender` for another SDK that runs in the same process, use that one.
```
Comment (haosong) [inline README.md:170]: @shavidissa Can you take a look at this `share-a-wavefront-sender` section? It used to be here: https://github.com/wavefrontHQ/wavefront-sdk-python/blob/3e61076489/docs/sender.md#share-a-wavefront-sender

Comment (shavidissa) [reply README.md:170]: Hi @haosong  and @jhmcstanton,

Thank you for picking this up! The correct URL is: https://github.com/wavefrontHQ/wavefront-sdk-doc-sources/blob/master/python/wavefrontsender.md#share-a-wavefront-sender. 

Do you think you could update the PR with this URL, please? :)

File: README.md [hunk 2]
```diff
@@ -167,9 +167,9 @@ See [Instantiating ApplicationTags](https://github.com/wavefrontHQ/wavefront-sdk
...
 
-* Otherwise, [Set Up a Wavefront Sender](https://github.com/wavefrontHQ/wavefront-sdk-python/blob/master/docs/sender.md#set-up-a-wavefront-sender).
+* Otherwise, [Set Up a Wavefront Sender](https://github.com/wavefrontHQ/wavefront-sdk-python#set-up-a-wavefront-sender).
```
Comment (shavidissa) [inline README.md:172]: Thanks again! The correct URL needs to be: https://github.com/wavefrontHQ/wavefront-sdk-doc-sources/blob/master/python/wavefrontsender.md#set-up-a-wavefront-sender

Comment (jhmcstanton) [reply README.md]: oh woops, corrected

File: README.md (hunk 1 above)
Comment (jhmcstanton) [reply README.md]: Thanks for the link, added :D

**Review by shavidissa — APPROVED** [review]
Thanks a lot for contributing, @jhmcstanton! :) LGTM!

**Comment by shavidissa** [comment]
Oh @jhmcstanton, for us to be able to merge the PR. You need to increment the version here: https://github.com/wavefrontHQ/wavefront-opentracing-sdk-python/blob/master/setup.py#L19. Can you update it to `2.2.1`, please?

@haosong, can you please confirm the above change? :)

**Comment by jhmcstanton** [comment]
Updated

**Comment by shavidissa** [comment]
Thanks again!

**Review by sushantdewan123 — APPROVED** [review]

**Pull request merged**